package/cpp/api/JsiSkAnimatedImageFactory.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <memory>
+#include <utility>
+
+#include <jsi/jsi.h>
+
+#include "JsiPromises.h"
+#include "JsiSkData.h"
+#include "JsiSkHostObjects.h"
+#include "JsiSkAnimatedImage.h"
+
+namespace RNSkia {
+
+namespace jsi = facebook::jsi;
+
+class JsiSkAnimatedImageFactory : public JsiSkHostObject {
+public:
+  JSI_HOST_FUNCTION(MakeAnimatedImageFromEncoded) {
+    auto data = JsiSkData::fromValue(runtime, arguments[0]);
+    auto codec = SkAndroidCodec::MakeFromData(data);
```

Comment:
so SkAndroidCodec works on all platforms except android? 🤔

Reply:
It seems to work when using the require syntax `useAnimatedImage(require("../../assets/birdFlying.gif"));` but it fails to decode when passing a base64 string directly on Android.